content/tutorials/r_brms/brms_nl/workshop_1_mcmc_en_brms.Rmd

```diff
@@ -1046,10 +1049,186 @@ comp_waic %>%
          ul_diff = elpd_diff  + qnorm(0.95) * se_diff)
 ```
 
-## Conclusie
+### Conclusie
 
 Zowel o.b.v. de PPC als vergelijkingen met verschillende model selectie criteria, kunnen we besluiten dat het tweede Poisson model met random intercepts het best past bij de data. In principe konden we dit ook verwachten op basis van onze eigen intuïtie en het design van de studie, nl. het gebruik van de Poisson distributie om aantallen te modelleren en het gebruik van random intercepts om te controleren voor een hiërarchisch design (habitats genest in sites).
 
+
+## Deep Dive: `rstan`
+### Stan: Wat? Waarom?!
+De `brms` package is een gemaksverpakking van `rstan`, dat zelf de functionaliteit van `stan` naar R brengt.
+Stan is een modeleringsomgeving, geschreven in de programmeertaal `C`, die de benodigdheden probabilistische ("Bayesiaanse") modellen implementeerd.
+
+Meer info vind je [op het internet](https://mc-stan.org).
+
+Het voordeel van `brms` is bruikbaarheid: veel functies werken "out-of-the-box", met redelijke standaardwaarden en een syntaxis die lijkt op wat frequentisten gewend zijn.
```

Comment:
niet enkel frequentisten, het is de gangbare syntax voor veel packages in R (zowel frequentistische als bayesiaanse) die met statistische modellen werken

Reply:
Klopt! I generalized to "statisticians".